app/api/run/route.ts

```diff
@@ -194,11 +194,13 @@ export async function DELETE(req: NextRequest) {
   }
 
   const data = await req.json();
-  const { id } = data;
+  const { runIds } = data;
 
-  await prisma.run.delete({
+  await prisma.run.deleteMany({
     where: {
-      id,
+      runId: {
+        in: runIds,
+      },
```

Comment:
Ensure that you only delete the intended `runIds` by also considering projectIds/taskIds or any other required columns in the where condition.